MAINTAINERS.md

```diff
@@ -0,0 +1,2 @@
+Please see
+(https://github.com/vmware-tanzu/carvel/blob/develop/MAINTAINERS.md)[https://github.com/vmware-tanzu/carvel/blob/develop/MAINTAINERS.md]
```

Comment:
```suggestion
Please see
https://github.com/vmware-tanzu/carvel/blob/develop/MAINTAINERS.md
```

Reply:
@danielhelfand will markdown still render an html link from just a raw url?

Reply:
Yes, it should